corehq/apps/case_search/xpath_functions/query_functions.py

```diff
@@ -69,6 +69,17 @@ def within_distance(node, context):
     return case_property_geo_distance(property_name, geo_point, **{unit: distance})
 
 
+def fuzzy_match(node, context):
+    """fuzzy-match(alias, 'pinky')"""
+    from corehq.apps.case_search.dsl_utils import unwrap_value
+
+    confirm_args_count(node, 2)
+    property_name = _property_name_to_string(node.args[0], node)
```

Comment:
so this still doesn't work for indexed cases, is that correct? It seems like `fuzzy-match(parent/alias, 'pinky')` wouldn't work here. I think that's fine if that is a known limitation.

Reply:
Correct it doesn't work for ancestor queries. I debated about making that work but decided it was out of scope and I'd rather add a `supercase-*` set of functions to match the `subcase-*` functions which would allow more flexible supercase queries.

(Not sure about the name 'supercase').

Reply:
🦸🏽